caching_5d289bae042863478674d6cd.md

```diff
@@ -0,0 +1,217 @@
+## cache
+
+Semaphore `cache` tool helps optimize CI/CD runtime by reusing files that your
+project depends on but are not part of version control. You should typically
+use caching to:
+
+- Reuse your project's dependencies, so that Semaphore fetches and installs
+them only when the dependency list changes.
+- Propagate a file from one block to the next.
+
+The cache is created on a per-project basis and available in every pipeline job.
+All cache keys are scoped per-project. The total cache size is 9.6GB.
+
+The `cache` tool uses key-path pairs for managing cached archives. An archive
+can be a single file or a directory.
+
+- [Basic Usage](#basic-usage)
+- [Advanced Usage](#advanced-usage)
+- [Cache Dependencies](#cache-dependencies)
+
+### Basic usage
+
```

Comment:
Would be great if we could explain with one or two paragraphs what does the basic usage means.

To clarify that it uses common paths for your dependancies to look up and store project dependancies in cache.